### DISCUSSION TITLE
Add a new function to Youtube ReVanced

### DISCUSSION THREAD

**Discussion by gallenjack** [community/community#53085] [Mobile] [answered]
### Select Topic Area

Question

### Body

Could you add a feature to completely block individual videos? So that even when you search, these videos won't show up.

**Comment by lizelive** [comment]
you are lost. interesting project, unsure what the correct repo is

https://github.com/j-hc/revanced-magisk-module/
might be it?
https://github.com/reisxd/revanced-builder/
is also valid

**Reply by piradata** [reply]
I think its this one?

https://github.com/revanced

**Reply by gallenjack** [reply]
I don't understand. Did I write my question in the wrong section? I followed your links but couldn't find where to write my question there 😢

**Reply by halotroop2288** [reply, marked as answer]
As someone who uses and has contributed to ReVanced before:

The issue should be raised on this repo.
http://github.com/revanced/revanced-patches

**Reply by piradata** [reply]
you can also talk with them on discord, they are very friendly

**Reply by pisssed** [reply]
Discord is where I used to buy my drugs from

**Comment by gallenjack** [comment]
Thank you! I wrote to them on the website, but they just closed my appeal
without a reply.

https://github.com/revanced/revanced-discussions/discussions/965

чт, 20 апр. 2023 г., 10:33 PIRADATA ***@***.***>:

**Comment by pisssed** [comment]
Maybe we should delete and say fuckit I just stopped a crime

**Reply by pisssed** [reply]
Just seems weird

**Comment by Kanthasuwamy-Kumaran** [comment]
thank you

**Comment by Ai1972** [comment]
Question

**Reply by Ai1972** [reply]
Label
